Sources/View+Buffer.swift

```diff
@@ -11,11 +11,14 @@ extension View {
     func makeBuffer(renderer: ImageRenderer<some View>) async throws -> CMSampleBuffer {
         // Pixel Buffer
         var buffer: CVPixelBuffer?
+        let scale = await UIScreen.main.scale * 2
```

Comment:
Out of curiosity, why do we multiply by two?

Makes absolute sense to use the screen scale here as most are 3x.

Reply:
Good question. I tried `UIScreen.main.scale` but it still looks low resolution in some cases. I chose 2x because it gives better results.  In the example below you can see 3x vs 6x. If 3x is acceptable we can update the code to 3x

![3x](https://github.com/user-attachments/assets/3072f12d-fbaa-4d5c-9038-af4846e34690)
![6x](https://github.com/user-attachments/assets/d2bd23eb-8186-4d23-96d8-796900fdd9fd)

Reply:
6x is definitely a **tonne** sharper. Happy to roll with it. Thanks for providing screenshots, super helpful!